tck/src/main/java/org/eclipse/configjsr/SimpleValuesBean.java

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Is already an inner class of the test it belongs to. No need to polute thei overall directory imo.